pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -72,16 +72,29 @@ func New(configVarResolver *providerconfig.ConfigVarResolver) cloudprovidertypes
 }
 
 type Config struct {
-	Kubeconfig       string
-	RestConfig       *rest.Config
-	DNSConfig        *corev1.PodDNSConfig
-	DNSPolicy        corev1.DNSPolicy
-	CPUs             string
-	Memory           string
-	Namespace        string
-	SourceURL        string
+	Kubeconfig         string
+	RestConfig         *rest.Config
+	DNSConfig          *corev1.PodDNSConfig
+	DNSPolicy          corev1.DNSPolicy
+	CPUs               string
+	Memory             string
+	Namespace          string
```

Comment:
As we will merge https://github.com/kubermatic/machine-controller/pull/1204 in the same release, we should now remove the `Namespace`.

Reply:
I find it confusing to keep the `Namespace` here as it's not a configurable option any more..